locale/en/manager.po

```diff
@@ -1439,6 +1439,53 @@ msgstr "Assign {$name} as {$role}"
 msgid "manager.categories.form.assignEditors.description"
 msgstr "Select the editorial users who should be assigned automatically to all new submissions to this category."
 
+msgid "manager.category.saved"
+msgstr "Category saved"
+
+msgid "manager.category.deleteCategory"
+msgstr "Delete Category"
+
+msgid "manager.category.confirmDelete"
+msgstr "I understand the consequences, delete this category"
+
+msgid "manager.category.delete.confirmationTitle"
+msgstr "Are you absolutely sure you want to delete \"{$title}\" category?"
+
+msgid "manager.category.delete.message.cannotBeUndone"
+msgstr ""
+"<div class=\"font-semibold\">"
+"Warning: Deleting this category will remove all {$subCategoryCount} sub-categories within it."
+"</div>"
+"<div class=\"text-secondary\">"
+"<p class=\"mt-6\">This action cannot be undone. Deleting the category will:</p>"
+"<ul role=\"list\" class=\"list-disc ml-8\">"
+"<li>Permanently remove all nested sub-categories</li>"
+"<li>Unassign this category from any submissions currently using it</li>"
+"</ul>"
+"<p class=\"mt-4\">This will not delete the submissions themselves — they will simply be left without a category.</p>"
+"</div>"
+"<p class=\"mt-4 text-primary text-lg-bold\">To confirm, please type the name of the category \"{$title}\" below to proceed</p>"
```

Comment:
I decided to add the styling and formatting here to avoid having to split these across multiple locale entries then styling them individually on the frontend.